bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/test_env/BigtableEnv.java

```diff
@@ -81,6 +81,29 @@ protected void setup() throws IOException {
       }
     }
 
+    // Configure DirectPath settings:
+    // - when a required mode is specified, ensure that ip address match the mode
+    // - specify a user agent that will trigger deny/allow lists on the serverside
+    switch (getConnectionMode()) {
+      case DEFAULT:
+        // nothing special
+        break;
+      case REQUIRE_CFE:
+        configuration.set("google.bigtable.test.data.ip.regex", "^(?!2001:4860:8040|34\\.126).*");
```

Comment:
nit: Should we use `BigtableOptionsFactory.BIGTABLE_TEST_DATA_IP_REGEX` and `CUSTOM_USER_AGENT_KEY ` here instead of "google.bigtable.test.data.ip.regex" and "google.bigtable.custom.user.agent"?

Reply:
Common doesn’t know about bigtable-hbase…it uses hbase apis only